### PR TITLE
Feature/crazyflie demo

### PR DESCRIPTION
**Pull request by BorjaOuterelo**

### CODE REVIEW

**Comment by ralph-lange** [comment]
I can reproduce the bug in the menu. I'll investigate it.

**Comment by ralph-lange** [comment]
@BorjaOuterelo, run Jekyll without the `--incremental` option or delete all build files (cf. .`gitignore`). It seems to be a caching issue.

**Comment by BorjaOuterelo** [comment]
@ralph-lange Yes in my side I deleted all build files and that worked.

File: _docs/tutorials/demos/crazyflie_demo/index.md [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+- its own attitude as a 3D vector (pitch, roll and yaw) using a *geometry_msg/Point32* message type on */drone/attitude* topic.
+
+The **Kobuki Turtlebot 2** robot is controlled using a UART protocol through a custom DB25 connector. The micro-ROS node runs on an Olimex STM32-E407 board attached to that UART port. This hardware features a [ST STM32F407](https://www.st.com/en/microcontrollers-microprocessors/stm32f407-417.html) MCU running **[Nuttx](https://nuttx.org/)** RTOS. In the same way, this node can communicate with the robot (UART) and with the ROS2 world (integrated Ethernet). Its used topics are:
+- a subscription on */cmd_vel* topic (*geometry_msg/Twist* message type) to receive the controlling angular and linear velocity.
+- a publication on */robot_pose* topic (*geometry_msg/Vector3* message type) which includes X position, Y position and robot yaw.
+
+The **external ROS 2 nodes** are rclpy tools with some different functionalities:
+- *attitude_to_vel.py*
+ - Converts Crazyflie */drone/attitude* to Kobuki Turtlebot 2 */cmd_vel* so that drone pitch is mapped to robot linear velocity and drone roll to angular valocity.
```
Comment (ralph-lange) [inline _docs/tutorials/demos/crazyflie_demo/index.md]: I propose to indent this item one level more or to directly put it after attitude_to_vel.py.

File: _docs/tutorials/demos/crazyflie_demo/index.md [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+ - Converts Crazyflie publications on */drone/attitude* and */drone/attitude* topics to *tf2_msgs/TFMessage* messages (required by RVIZ visualizer)
+- *odom_to_tf.py*
+ - Converts Kobuki Turtlebot 2 publications on */robot_pose* topic to *tf2_msgs/TFMessage* messages (required by RVIZ visualizer).
```
Comment (ralph-lange) [inline _docs/tutorials/demos/crazyflie_demo/index.md]: I propose to indent this item one level more or to put it directly, i.e. in one line, after odom_to_tf.py.

Comment (BorjaOuterelo) [reply _docs/tutorials/demos/crazyflie_demo/index.md]: Solved in 9af6534

File: _docs/tutorials/demos/crazyflie_demo/index.md (hunk 1 above)
Comment (BorjaOuterelo) [reply _docs/tutorials/demos/crazyflie_demo/index.md]: Solved in 9af6534

**Pull request merged**